### PR TITLE
Fixed exceptions thrown in behat scenarios

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

File: Behat/Context/Page/Element/Grid.php [hunk 1]
```diff
@@ -59,12 +59,12 @@ public function getCell($rowPosition, $columnPosition)
     {
         $row = $this->find('xpath', sprintf('//tbody/tr[%d]', $rowPosition));
         if (null === $row) {
-            throw new BehaviorException(sprintf('Unable to find row %d', $rowPosition));
+            throw new UnexpectedPageException(sprintf('Unable to find row %d', $rowPosition));
```
Comment (rn0) [inline Behat/Context/Page/Element/Grid.php]: Semantically better would be `SensioLabs\Behat\PageObjectExtension\PageObject\Exception\ElementNotFoundException`

**Comment by JarekW** [comment]
:+1:

**Comment by szymach** [comment]
:+1:

**Comment by rn0** [comment]
:+1:

**Pull request merged**